src/docs/sdk/research/performance/index.mdx

```diff
@@ -192,6 +192,38 @@ In the example above, if an error bubbles up the call stack we want to be able t
 
 All those different expectations makes it hard to reuse, in an understandable way, the current notion of `scope`, how breadcrumbs are recorded, and how those different concepts interact.
 
+Finally, it is worth noting that the changes to restructure scope management most likely cannot be done without breaking existing SDK APIs. Existing SDK concepts, like hubs, scopes, breadcrumbs, user, tags, contexts, would all have to be remodeled.
+
 ## Span Ingestion Model
 
-Coming soon.
+Consider a trace depicted by the following span tree:
+
+```
+F
+├─ B*
+│  ├─ B
+│  ├─ B
+│  ├─ B
+│  │  ├─ S*
+│  │  ├─ S*
+│  ├─ B
+│  ├─ B
+│  │  ├─ S*
+│  ├─ B
+│  ├─ B
+│  ├─ B
+│  │  ├─ S*
+
+where
+F: span created on frontend service
+B: span created on backend service
+S: span created on storage service
+```
+
+This trace illustrates 3 services instrumented such that a user clicks a button on a Web page (`F`) and a backend (`B`) performs some work which involves making several queries to a storage service (`S`). Spans that are at the entry point to a given service are marked with a `*` to denote that they are transactions.
+
+Now we can use this example to compare and understand the difference between Sentry's span ingestion model and the model used by OpenTelemetry and other similar tracing systems.
+
+In Sentry's span ingestion model, all spans that belong to a transaction must be sent all together in a single request. That means that all `B` spans must be kept in-memory for the whole duration of the `B*` transaction, including time spent on downstream services (the storage service in the example).
```

Comment:
Can we make it explicit that this is because span data cannot be sent to Sentry outside of a transaction. Feel like that can be worded better, but I want to make sure that people understand that transaction events -> only way to group and ingest performance data right now.

Reply:
Open to rewording suggestions :)

Reply:
I think this will be address by at least two of the problems related to the ingestion model:
- Spans Cannot Exist Outside of a Transaction
- Transaction Spans Must Be Converted to Transaction Events

The example should set the stage, and then we start listing out and exploring the identified issues.

Reply:
> I think this will be addressed by at least two of the problems related to the ingestion model:

Agree, let's hold off on my suggestion then.